gpic/gravatar.go

```diff
@@ -35,6 +38,10 @@ type Image struct {
 	disableDefault	bool
 }
 
+type GithubUser struct {
```

Comment:
Some thoughts here:

- This is called GithubUser but this in a file called `gravatar.go`. Either the file name should be changed or this should be broken out to its own file.
- This doesn't represent a GitHub user.
- Like the struct for Gravatar, we should likely break this URL out into its components. Not just an entire URL. It makes it harder to modify the components when its in this form.